README.md

```
@@ -80,17 +80,17 @@ Built with Kotlin Multiplatform and Compose Multiplatform for Android and Deskto
 </div>
 
 <p align="center">
-  <a href="https://github.com/OpenHub-Store/GitHub-Store/releases">
+  <a href="https://github.com/kurikomi-labs/komi-store/releases">
     <img src="https://i.ibb.co/q0mdc4Z/get-it-on-github.png" height="80" />
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)